target/linux/realtek/dts-5.10/rtl9302_zyxel_xgs1210-12.dts

```diff
@@ -0,0 +1,326 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/dts-v1/;
+
+#include "rtl930x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "zyxel,xgs1210-12", "realtek,rtl838x-soc";
+	model = "Zyxel XGS1210-12 Switch";
+
+	// mw.l 0xb8003308 0x00200000 active low toggles PHY reset
```

Comment:
I suppose that wasn't to be there?

```suggestion
```

Reply:
No, especially because then people want you to put a PHY-Reset property somewhere...
This was more to keep track of GPIOs. Currently I know:
`
Board GPIO
============================
Device  Pin  Direction  Default  Current
------- ---- ---------- -------- --------
EXT     8    OUT (OUT ) 0        1       LED: Power
EXT     10   OUT (OUT ) 1        1       LED: Mode
INT     22   IN         1        1       BUTTON: Led-Mode
INT     9    IN         1        1       BUTTON: Reset
INT     12   IN         1        1       BUTTON: Restore
INT     23   OUT        1        1       
EXT     23   OUT (OUT ) 1        1       RESET
EXT     24   OUT (OUT ) 1        1       
EXT     25   OUT (OUT ) 1        1       MAX3421: RESET, active low
INT     15   OUT        0        0       
INT     16   OUT        0        0       
INT     17   IN         0        0       
INT     18   OUT        0        0       
INT     19   IN         0        1      MAX3421: IRQ 
INT     8    OUT        0        0       
INT     13   OUT        0        1      POE: SCL 
INT     14   OUT        0        1      POE: SDA
EXT     0    IN  (IN  ) 1        0      SFP: LOS P24       
EXT     1    IN  (IN  ) 1        1      SFP: LOS P25 
EXT     2    IN  (IN  ) 1        0      SFP: LOS P26
EXT     3    IN  (IN  ) 1        1      SFP: LOS P27
EXT     4    IN  (IN  ) 1        0      SFP: MOD-DEF0 P24
EXT     5    IN  (IN  ) 1        1      SFP: MOD-DEF0 P25
EXT     6    IN  (IN  ) 1        0      SFP: MOD-DEF0 P26
EXT     7    IN  (IN  ) 1        1      SFP: MOD-DEF0 P27
EXT     13   IN  (OUT ) 1        1      SFP: SDA P24
EXT     14   IN  (OUT ) 1        1      SFP: SDA P25       
EXT     21   IN  (OUT ) 1        1      SFP: SDA P26 
EXT     22   IN  (OUT ) 1        1      SFP: SDA P27 
EXT     29   OUT (OUT ) 1        1      SFP: SCL P24, P25       
EXT     30   OUT (OUT ) 1        1      SFP: SDA P26, P27 
EXT     19   IN  (OUT ) 1        1      FAN: SDA
EXT     20   OUT (OUT ) 1        1      FAN: SCL 
EXT     28   OUT (OUT ) 1        1      FAN: SPEED???
EXT     31   IN  (IN  ) 0        0       
EXT     32   IN  (IN  ) 0        1       
EXT     33   IN  (IN  ) 0        0
`

This will be removed as suggested.